packages/webchat/src/index.tsx

```diff
@@ -57,6 +57,9 @@ export { Embedded as embedded } from '.'
 export { Fullscreen as fullscreen } from '.'
 
 export * from './typings'
+
+// TODO: export this
+/*
```

Comment:
Why not simply remove the comments then?

Reply:
This is a reminder for the future, we'll keep it here for now but it'll be removed/uncommented eventually.

Reply:
Alright, then what's the goal of having those exported?